web/angular-wallet/src/app/main/transactions/transaction.service.ts

```diff
@@ -42,8 +44,11 @@ export class TransactionService {
 
     public currentAccount: BehaviorSubject<any> = new BehaviorSubject(undefined);
 
-    constructor(apiService: ApiService, private accountService: AccountService) {
+    constructor(apiService: ApiService, private accountService: AccountService, private storeService: StoreService) {
         this.transactionApi = apiService.api.transaction;
+        this.storeService.settings.subscribe((settings: Settings) => {
```

Comment:
I always use the unsubscribe pattern...

Reply:
This is a service, e.g. a singleton. I don't think we unsubscribe from these, see https://github.com/burst-apps-team/phoenix/blob/develop/web/angular-wallet/src/app/setup/account/account.service.ts#L69

Reply:
Yeah... You're right